chat-plugins/mafia.js

```diff
@@ -1197,13 +1210,14 @@ exports.commands = {
 			target = target.split(',');
 			if (Rooms(target[0]) && Rooms(target[0]).users[user.userid]) targetRoom = Rooms(target.shift());
 			if (!targetRoom || !targetRoom.game || targetRoom.game.gameid !== 'mafia') return this.errorReply(`There is no game of mafia running in this room.`);
-			if (!user.can('mute', null, room) && targetRoom.game.hostid !== user.userid) return user.sendTo(targetRoom, `|error|/mafia deadline - Access denied.`);
+			if (!user.can('mute', null, room) && targetRoom.game.hostid !== user.userid && target) return user.sendTo(targetRoom, `|error|/mafia deadline - Access denied.`);
 			target = toId(target.join(''));
 			if (target === 'off') {
 				return targetRoom.game.setDeadline('off');
 			} else {
 				target = parseInt(target);
 				if (isNaN(target)) {
+					if (room.game.hostid === user.userid && this.cmdToken === "!") this.broadcasting = true;
 					if (!this.runBroadcast()) return;
```

Comment:
Update `room.lastBroadcast` and `room.lastBroadcastTime`

Or actually maybe it would be a good idea to allow games to modify who can broadcast without promoting them?
`game.canBroadcast` maybe.. Thoughts @Zarel?